CHANGELOG.md

```diff
@@ -1,3 +1,191 @@
+	[#1241](https://github.com/weaveworks/scope/pull/1241)
+	[#1209](https://github.com/weaveworks/scope/pull/1209)
+	[#1232](https://github.com/weaveworks/scope/pull/1232)
+
+
+Performance improvements:
+- UI performance improvements
+  - Performance improvements for canvas
+		[#1186](https://github.com/weaveworks/scope/pull/1186)
+		[#1236](https://github.com/weaveworks/scope/pull/1236)
+		[#1239](https://github.com/weaveworks/scope/pull/1239)
+		[#1262](https://github.com/weaveworks/scope/pull/1262)
+  - Reduce CPU consumption if UI cannot connect to backend
+		[#1229](https://github.com/weaveworks/scope/pull/1229)
+  - Cache pan/zoom per topology
+		[#1261](https://github.com/weaveworks/scope/pull/1261)
+
+
+Bug Fixes:
+- Scope app doesn't correctly expire old reports
+	[#1286](https://github.com/weaveworks/scope/issues/1286)
+- Container nodes appear without a host label
+	[#1065](https://github.com/weaveworks/scope/issues/1065)
+- Resizing the window and zooming in/out can confuse window size
+	[#1180](https://github.com/weaveworks/scope/issues/1096)
+- Link from container -> Pod doesn't work
+    [#1180](https://github.com/weaveworks/scope/issues/1293)
+- Various websocket and pipe fixes.
+	[#1172](https://github.com/weaveworks/scope/pull/1172)
+	[#1175](https://github.com/weaveworks/scope/pull/1175)
+- Make `--app-only` only run the app and not probe
+	[#1067](https://github.com/weaveworks/scope/pull/1067)
+- Exported SVG file throws "CANT" error in Adobe Illustrator
+	[#1144](https://github.com/weaveworks/scope/issues/1144)
+- Docker labels not rendering correctly
+	[#1284](https://github.com/weaveworks/scope/issues/1284)
+- Error when parsing kernel version in `/proc` background reader
+	[#1136](https://github.com/weaveworks/scope/issues/1136)
+- Opening the terminal doesn't open work for some containers
+	[#1195](https://github.com/weaveworks/scope/issues/1195)
+- Terminals: Try to figure what shell to use instead of simply running `/sh/bin`
+	[#1069](https://github.com/weaveworks/scope/pull/1069)
+- Fix embedded logo size for Safari
+	[#1084](https://github.com/weaveworks/scope/pull/1084)
+- Don't read from app.Version before we initialise it
+	[#1163](https://github.com/weaveworks/scope/pull/1163)
+- Hotkeys for metrics are not working in Firefox and are hard to find
+	[#1228](https://github.com/weaveworks/scope/issues/1228)
+- Focus makes nodes huge; previously it shrank other nodes
+	[#1244](https://github.com/weaveworks/scope/issues/1244)
+- Don't show multiple pseudo nodes in the host view for the same IP
+	[#1155](https://github.com/weaveworks/scope/issues/1155)
+- Fix race conditions detected by race detector from Go 1.6
+	[#1192](https://github.com/weaveworks/scope/issues/1192)
+	[#1087](https://github.com/weaveworks/scope/issues/1087)
+
+
+Documentation:
+- Provide Docker Compose examples for launching the Scope probe with the Scope Cloud Service
+	[#1146](https://github.com/weaveworks/scope/pull/1146)
+
+Experimental features:
```

Comment:
I would be tempted to drop this + internal improvements section.  We've not included them in the past.